x/cellarfees/types/keys.go

```diff
@@ -13,3 +13,12 @@ const (
 	// QuerierRoute is the querier route for cellarfees
 	QuerierRoute = ModuleName
 )
+
+var (
+	// key for global cellar fee pool state
+	CellarFeePoolKey = []byte{0x00}
+	// key for storing the reward supply after the latest increase
+	LastHighestRewardSupply = []byte{0x01}
+	// key for storing the next scheduled auction height
+	ScheduledAuctionHeight = []byte{0x02}
+)
```

Comment:
@EricBolten can correct me but I think we need to skip the 0 byte index. Can't recall the reasoning off the top of my head

Might be nice to also make these keys constant instead, and use Get methods to actually retrieve the keys. E.g.

```
const (
	_ = byte(iota)

	// key for global cellar fee pool state
	CellarFeePoolKey 

	// key for storing the reward supply after the latest increase
	LastHighestRewardSupplyKey

	// key for storing the next scheduled auction height
	ScheduledAuctionHeightKey
)

// GetCellarFeePoolKey returns the key prefix 
func GetCellarFeePoolKey() []byte {
	return []byte{CellarFeePoolKey}
}

// GetLastHighestRewardSupplyKey returns the key prefix 
func GetLastHighestRewardSupplyKey() []byte {
	return []byte{LastHighestRewardSupplyKey}
}

// GetScheduledAuctionHeightKey returns the key prefix 
func GetScheduledAuctionHeightKey() []byte {
	return []byte{ScheduledAuctionHeightKey}
}

```

Reply:
I'm not sure if it's totally necessary to skip the zero index, but that was the pattern used by the other keys files in our modules, using the `_ = byte(iota)` pattern. I agree with the `const` and getter pattern.

The downside to the existing method is that it's kind of brittle in the sense that the byte values are determined by the order of elements in code, so an accidental change could muck things up significantly, in the same way as e.g. renumbering a proto's ID fields.